Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -699,11 +699,14 @@ class SettingsTableSectionHeaderFooterView: UITableViewHeaderFooterView {
 
     override init(reuseIdentifier: String?) {
         super.init(reuseIdentifier: reuseIdentifier)
-        contentView.backgroundColor = SettingsUX.TableViewHeaderBackgroundColor
+        backgroundView = UIView()
+        backgroundView!.backgroundColor = SettingsUX.TableViewHeaderBackgroundColor
```

Comment:
even though the `backgroundView` was created in the line above this. We usually try to avoid force unwrapping